tests/system/src/test/java/test/robot/javafx/scene/PixelBufferDrawTest.java

```diff
@@ -186,10 +186,10 @@ private ImageView createImageViewPB(PixelBuffer<? extends Buffer> pixelBuffer) {
     }
 
     private void compareColor(Color exp, Color act) {
-        Assert.assertEquals(exp.getRed(), act.getRed(), 0.005);
-        Assert.assertEquals(exp.getBlue(), act.getBlue(), 0.005);
-        Assert.assertEquals(exp.getGreen(), act.getGreen(), 0.005);
-        Assert.assertEquals(exp.getOpacity(), act.getOpacity(), 0.005);
+        Assert.assertEquals(exp.getRed(), act.getRed(), 0.01);
```

Comment:
Since the tolerance value is used 4 times, maybe consider using a variable? Since this is preexisting, I'll approve it anyway and leave it up to you as to whether to change it.

Reply:
Since someone initially chose a value w/o justification, perhaps we should create a constant and describe it (using Kevin's comment as a starting point).

Reply:
I will update it.